### ISSUE TITLE
autograph=False not respected when calling get_concrete_function()

### ISSUE THREAD

**Issue by t-kalinowski** [tensorflow/tensorflow#32680]
**System information**
- Have I written custom code (as opposed to using a stock example script provided in TensorFlow): Yes
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): reproduced on Ubuntu 18.04 and MacOS 10.14
- TensorFlow installed from (source or binary): binary
- TensorFlow version (use command below): 1.14.0  v1.14.0-rc1-22-gaf24dc91b5
- Python version: 2.7.16

**Describe the current behavior**
When passing in `tf.function(..., autograph=False)`, console output indicates that `autograph` still attempts to autograph the function.

**Describe the expected behavior**
That the function is not going to be autographed.

**Code to reproduce the issue**
This encountered when passing in a function that `autograph` fails on. For example, a python function that is actually a wrapper around an R function.

This example is in R:
```R

> library(reticulate)
> library(tensorflow)
> 
> Sys.setenv(AUTOGRAPH_VERBOSITY=10)
> reticulate::use_virtualenv("~/tf1")
> 
> x <- tf$constant(1)
> add1 <- py_func(function() x + 1)
> 
> fn <- tf$`function`(add1, autograph = FALSE)
> 
> fn$get_concrete_function()
ERROR:tensorflow:Error converting <function initialize_variables at 0x134c82140>
Traceback (most recent call last):
  File "/Users/tomasz/tf1/lib/python2.7/site-packages/tensorflow/python/autograph/impl/api.py", line 524, in to_graph
    return conversion.convert(entity, program_ctx)
  File "/Users/tomasz/tf1/lib/python2.7/site-packages/tensorflow/python/autograph/impl/conversion.py", line 306, in convert
    entity, program_ctx, free_nonglobal_var_names)
  File "/Users/tomasz/tf1/lib/python2.7/site-packages/tensorflow/python/autograph/impl/conversion.py", line 229, in _convert_with_cache
    entity, program_ctx)
  File "/Users/tomasz/tf1/lib/python2.7/site-packages/tensorflow/python/autograph/impl/conversion.py", line 431, in convert_entity_to_ast
    nodes, name, entity_info = convert_func_to_ast(o, program_ctx)
  File "/Users/tomasz/tf1/lib/python2.7/site-packages/tensorflow/python/autograph/impl/conversion.py", line 624, in convert_func_to_ast
    node = node_to_graph(node, context)
  File "/Users/tomasz/tf1/lib/python2.7/site-packages/tensorflow/python/autograph/impl/conversion.py", line 657, in node_to_graph
    node = converter.standard_analysis(node, context, is_initial=True)
  File "/Users/tomasz/tf1/lib/python2.7/site-packages/tensorflow/python/autograph/core/converter.py", line 354, in standard_analysis
    node = qual_names.resolve(node)
  File "/Users/tomasz/tf1/lib/python2.7/site-packages/tensorflow/python/autograph/pyct/qual_names.py", line 254, in resolve
    return QnResolver().visit(node)
  File "/usr/local/opt/python@2/Frameworks/Python.framework/Versions/2.7/lib/python2.7/ast.py", line 241, in visit
    return visitor(node)
  File "/usr/local/opt/python@2/Frameworks/Python.framework/Versions/2.7/lib/python2.7/ast.py", line 297, in generic_visit
    value = self.visit(value)
  File "/usr/local/opt/python@2/Frameworks/Python.framework/Versions/2.7/lib/python2.7/ast.py", line 241, in visit
    return visitor(node)
  File "/usr/local/opt/python@2/Frameworks/Python.framework/Versions/2.7/lib/python2.7/ast.py", line 297, in generic_visit
    value = self.visit(value)
  File "/usr/local/opt/python@2/Frameworks/Python.framework/Versions/2.7/lib/python2.7/ast.py", line 241, in visit
    return visitor(node)
  File "/usr/local/opt/python@2/Frameworks/Python.framework/Versions/2.7/lib/python2.7/ast.py", line 306, in generic_visit
    new_node = self.visit(old_value)
  File "/usr/local/opt/python@2/Frameworks/Python.framework/Versions/2.7/lib/python2.7/ast.py", line 241, in visit
    return visitor(node)
  File "/usr/local/opt/python@2/Frameworks/Python.framework/Versions/2.7/lib/python2.7/ast.py", line 297, in generic_visit
    value = self.visit(value)
  File "/usr/local/opt/python@2/Frameworks/Python.framework/Versions/2.7/lib/python2.7/ast.py", line 241, in visit
    return visitor(node)
  File "/Users/tomasz/tf1/lib/python2.7/site-packages/tensorflow/python/autograph/pyct/qual_names.py", line 236, in visit_Subscript
    if isinstance(s.value, gast.Num):
AttributeError: 'module' object has no attribute 'Num'
WARNING:tensorflow:Entity <function initialize_variables at 0x134c82140> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <function initialize_variables at 0x134c82140>: AttributeError: 'module' object has no attribute 'Num'
<tensorflow.python.eager.function.ConcreteFunction>
```

**Comment by t-kalinowski** [comment]
I just checked against TensorFlow version 1.15.0rc1 and this issue is fixed there.

**Comment by t-kalinowski** [comment]
Opening back up because this is not fixed in TensorFlow version 2.0.0rc1
```R
> library(reticulate)
> library(tensorflow)
> 
> Sys.setenv(AUTOGRAPH_VERBOSITY=10)
> reticulate::use_virtualenv("~/tf2")
> 
> tf$version$VERSION
[1] "2.0.0-rc1"
> tf$version$GIT_VERSION
[1] "v2.0.0-rc0-101-gd2d2566eef"
> 
> x <- tf$constant(1)
2019-09-20 00:03:35.132646: I tensorflow/core/platform/cpu_feature_guard.cc:142] Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX2 FMA
2019-09-20 00:03:35.146449: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x7fc075f78db0 executing computations on platform Host. Devices:
2019-09-20 00:03:35.146465: I tensorflow/compiler/xla/service/service.cc:175]   StreamExecutor device (0): Host, Default Version
> add1 <- py_func(function() x + 1)
> 
> fn <- tf$`function`(add1, autograph = FALSE)
> 
> fn$get_concrete_function()
WARNING:tensorflow:Entity <function initialize_variables at 0x10adc18c0> could not be transformed and will be executed as-is. Please report this to the AutoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: 'module' object has no attribute 'Num'
<tensorflow.python.eager.function.ConcreteFunction>

```

**Comment by gadagashwini** [comment]
@t-kalinowski, 
I could reproduce the issue with Tf 2.0.0.rc1. Can you try with latest Tf 2.0 nightly version. Thanks!

**Comment by t-kalinowski** [comment]
Thanks,
This appears to be fixed in tf 2.0 nightly preview (tf version `2.0.0-dev20190920`) and in tf version 1.15.0-rc1 

However, this is currently broken in the tf v1 nightly (taken from `pip install tf-nightly`, tf version `1.15.0-dev20190821`

**Comment by ymodak** [comment]
TF 1.15 rc1 is the latest TF 1.X release so we can expect the fix to be included in final TF 1.15 version as well.
TF nightly version ```1.15.0-dev20190821``` was built on 08/21/2019 where as TF 1.15rc1 version was released on 09/16/2019.
